config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+get "/projects/:id", to: "projects#show"
+post "/projects/:id", to: "contestant_projects#new"
```

Comment:
Your controller organization is awesome, but your path for the POST could be made more RESTful if it included contestants. Right now, it looks like it's a path just for dealing with projects but really it's for creating an association between a given project and contestant. So a more RESTful path could be "/projects/:project_id/contestants"